Cargo.toml

```diff
@@ -11,6 +11,7 @@ keywords = ["gis", "geo", "geography", "geospatial"]
 
 [dependencies]
 num-traits = "0.1"
+cgmath = "^0.12.0"
```

Comment:
FYI `^` is redundant here. In a Cargo.toml file, `0.12.0` and `^0.12.0` are the same thing

http://doc.crates.io/specifying-dependencies.html#specifying-dependencies-from-cratesio